cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -1954,6 +1958,45 @@ String zkConnectionString(int connectToReplicas, Function<Integer, String> zkNod
             return withVoid(fut);
         }
 
+        /**
+         * Get the cluster Id of the Kafka cluster
+         * 
+         * @return
+         */
+        Future<ReconciliationState> kafkaGetClusterId() {
+            if (kafkaStatus.getClusterId() != null) {
+                // Cluster Id does not change
+                return Future.succeededFuture(this);
+            } else {
```

Comment:
The status starts empty every time ... so I would normally not expect this to every happen. Is it really needed?

Reply:
ok, I will get rid of it.